### PR TITLE
feat(sms): Add support for Austria, Germany 

### LINKED ISSUE #5572: Release SMS pt3 to Germany and Austria
SMS Phase 3 flow improved mobile pairing flows.

It is currently available in only a select few countries. This issue is to deploy it in Germany and Austria.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
fixes #5572

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
@mozilla/fxa-devs - r?

File: app/scripts/lib/country-telephone-info.js [hunk 1]
```diff
@@ -60,15 +60,41 @@ define((require, exports, module) => {
...
+    },
+    // Germany
+    // https://en.wikipedia.org/wiki/Telephone_numbers_in_Germany
+    DE: {
+      format: formatter('+49 ${serverPhoneNumber}'),
+      normalize: ensurePrefix('+49'),
+      pattern: /^(?:\+49)?\d{6,13}$/,
+      prefix: '+49',
+      rolloutRate: 0.5
+    },
```
Comment (philbooth) [inline app/scripts/lib/country-telephone-info.js]: I love how declarative these changes are! It's right up my cup of tea.

File: app/scripts/lib/country-telephone-info.js [hunk 2]
```diff
@@ -60,15 +60,41 @@ define((require, exports, module) => {
...
+
   module.exports = {
+    // Austria
+    // https://en.wikipedia.org/wiki/Telephone_numbers_in_Austria
+    AT: {
+      format: formatter('+43 ${serverPhoneNumber}'),
+      normalize: ensurePrefix('+43'),
+      pattern: /^(?:\+43)?\d{6,}$/,
+      prefix: '+43',
+      rolloutRate: 0.5
```
Comment (shane-tomlinson) [inline app/scripts/lib/country-telephone-info.js]: @davismtl - I know we discussed rolling this out to 50% of people, while discussing with @rfk today he suggested we could do a soft-launch, set the rate to 0, ask some community members to test once we release, and then bump the number the following train.

I think that's a reasonable suggestion.

Comment (davismtl) [reply app/scripts/lib/country-telephone-info.js]: I think that would be prudent.

What would be the best way to reach out to DE community members to test?

Comment (shane-tomlinson) [reply app/scripts/lib/country-telephone-info.js]: > What would be the best way to reach out to DE community members to test?

I know a few Mozillians in Germany that I can ask for help. I might need help with Austria though.

**Pull request merged**